pageserver/src/tenant/vectored_blob_io.rs

```diff
@@ -61,26 +61,36 @@ pub struct VectoredRead {
 }
 
 impl VectoredRead {
-    fn size(&self) -> usize {
+    pub(crate) fn size(&self) -> usize {
```

Comment:
`pub(crate)` instead of `pub` to get dead code warnings (pub means "is public API").